rules/code-quality/src/Rector/Foreach_/SimplifyForeachToCoalescingRector.php

```diff
@@ -152,10 +152,11 @@ private function processForeachNodeWithReturnInside(Foreach_ $foreach, Return_ $
         } else {
             return null;
         }
+        $foreachGetAttribute = $foreach->getAttribute(AttributeKey::NEXT_NODE);
```

Comment:
The variable name from `getAttribute()`, should be named after `AttributeKey::<NEXT_NODE>`.

Like this:

```php
$nextNode = $foreach->getAttribute(AttributeKey::NEXT_NODE);

$phpDocInfo = $foreach->getAttribute(AttributeKey::PHP_DOC_INFO);
```

etc.

Reply:
Btw, I recommend reverting the applied Rector commit (these changes above). So we work with original code

Reply:
@TomasVotruba do you mean we don't need addition to rector config set ?

Reply:
We do. Just the Rector commits, e.g. [`1a7d320` (#4457)](https://github.com/rectorphp/rector/pull/4457/commits/1a7d320fe3835515c4988110ca38bf276d88af31)

should be reverted before applied new version rule.


If you change the rule now, then it will not update any code, because it's been moved from `if()`.

If you revert the Rector commit, then change the rule, then the Rule will be applied in it's new form.